packages/create-pantheon-decoupled-kit/src/generators/gatsby-wp.generator.ts

```diff
@@ -0,0 +1,47 @@
+import type { CustomActionConfig } from 'node-plop';
+import type { DecoupledKitGenerator } from '../types';
+import whichPmRuns from 'which-pm-runs';
+
+export const gatsbyWp: DecoupledKitGenerator = {
+	name: 'gatsby-wp',
+	description: 'Gatsby + WordPress starter kit',
+	prompts: [
+		{
+			name: 'appName',
+			message: 'What is the name of your project?',
+			default: 'Gatsby WordPress Starter',
+		},
+		{
+			name: 'outDir',
+			message: 'Where should the output go?',
+			default: ({ appName }: { [key: string]: string }) =>
+				`${process.cwd()}/${appName.replaceAll(' ', '-').toLowerCase()}`,
+		},
+	],
+	actions: (data) => {
+		let pnpm = false;
+		const getPkgManager = whichPmRuns();
+		if (getPkgManager && getPkgManager.name === 'pnpm') {
+			pnpm = true;
+		}
```

Comment:
This could be a bit more succinct
```suggestion
		const pnpm = whichPmRuns()?.name === 'pnpm' ? true : false;

```